.github/workflows/ci.yml

```
@@ -0,0 +1,38 @@
+name: Python package
+
+on: [push, pull_request]
```

I am unsure about the syntax, but I think this should be

```suggestion
on:
    push:
    pull_request:
```